src/gt4py/next/program_processors/runners/dace_fieldview/gtir_builtin_translators.py

```diff
@@ -369,6 +369,89 @@ def translate_literal(
     return [(data_node, data_type)]
 
 
+def translate_scalar_expr(
+    node: gtir.Node,
+    sdfg: dace.SDFG,
+    state: dace.SDFGState,
+    sdfg_builder: gtir_to_sdfg.SDFGBuilder,
+    reduce_identity: Optional[gtir_to_tasklet.SymbolExpr],
+) -> list[TemporaryData]:
+    assert isinstance(node, gtir.FunCall)
+    assert isinstance(node.type, ts.ScalarType)
+
+    args = []
+    connectors = []
+    scalar_expr_args = []
+
+    for arg_expr in node.args:
+        is_symbol = True
+        if isinstance(arg_expr, gtir.SymRef):
+            try:
+                sdfg_builder.get_symbol_type(arg_expr.id)
+            except KeyError:
+                is_symbol = False
```

Comment:
A suggestion, feel free to ignore.
But Would it make sense to create a helper function to test this, i.e. `sdfg_builder.is_symbol()`.

Reply:
Good point. On the other hand, I was considering to add this additional check that anyway requires `get_symbol_type`:
```suggestion
        is_symbol = True
        if isinstance(arg_expr, gtir.SymRef):
            try:
                symbol_type = sdfg_builder.get_symbol_type(arg_expr.id)
                assert isinstance(symbol_type, ts.ScalarType)
            except KeyError:
                is_symbol = False
```